docs/sidebar.yml

```diff
@@ -188,8 +205,10 @@ subsection:
       - page: internals/dotc-scalac.md
       - page: internals/higher-kinded-v2.md
       - page: internals/overall-structure.md
+      - page: internals/explicit-nulls.md
       - page: internals/periods.md
       - page: internals/syntax.md
+      - page: internals/syntax-3.1.md
```

Comment:
Those files have similar titles, but also similar content. About the title, I think these files contain more than a summary, it looks like a specification, so I would say “Scala Syntax Specification”. There is also the file in `reference/syntax.md` that is very similar. @odersky, what could we add to the top of `internals/syntax.md` and `internals/syntax-3.1.md` to explain why they differ from each other, and why they differ from `reference/syntax.md`?